rfd/0080-hardware-key-support.md

```diff
@@ -105,6 +129,8 @@ We will start with the following private key policies:
     * In some uncommon cases, this pin caching is unreliable. If `tsh` gets a pin auth error when it expected the PIN to be cached in the connection, `tsh` will retry with the pin prompt and cache the pin for the remainder of the program execution. In these cases, PIN will always be prompted for once per `tsh` connection, but never twice.
   * This policy is intended for rare circumstances where a touch policy can not be configured due to the use of external PIV tools. However, since pin alone does not verify user presence, this option opens the door for remote attacks. When possible, `hardware_key_touch_and_pin` should be used instead of this option.
 * `hardware_key_touch_and_pin`: combination of `hardware_key_touch` and `hardware_key_pin`.
+* `web_session`: private key stored as a web session in the Auth service storage. This key is only accessible by the Auth and Proxy services. Keys with this policy meet all other key policy requirements.
+  * Currently there is no way to enforce this policy, but web sessions will have this policy in order to meet any other policy requirements.
```

Comment:
What I mean is that a cluster can't enforce `private_key_policy: web_session`. This is theoretically possible though, and it would make it so only the WebUI / web session could access the cluster, not `tsh` or Teleport Connect.

The point of the policy is that certificates that are signed with `private_key_policy: web_session` will meet the actual policy requirement of the cluster, Ex: `hardware_key_touch`.

But I think this is kind of an odd/insignificant point and is more of an implementation concern, I'll reword it or maybe just remove it.